src/frontend/js/order.js

```diff
@@ -23,4 +23,45 @@ $(function() {
             }
         });
     });
+
+    // Place batch orders
+    // --
+    // Init multidatepicker on input directly would be simpler,
+    // but a glitch would appear, so I init multidatespicker on an empty div#delivery_dates
+    // and link it to an HTML input#id_delivery_dates using altField option
+    // @see: https://github.com/dubrox/Multiple-Dates-Picker-for-jQuery-UI/issues/162
+    $('#delivery_dates').multiDatesPicker({
+        dateFormat: "yy-mm-dd",
+        separator: "|",
+        minDate: 0,
+        numberOfMonths: 2,
+        altField: '#id_delivery_dates'
+    });
+    $('#id_delivery_dates').hide();
+
+    $('#id_client').change(function() {
+        $.get('/member/client/' + $(this).val() + '/meals/preferences', function(data) {
```

Comment:
I created a different issue to fix this: https://github.com/savoirfairelinux/sous-chef/issues/494